### DISCUSSION TITLE
Read local storage or cookies in the extensions

### DISCUSSION THREAD

**Discussion by hkdeman** [coder/code-server#3603] [Q&A] [answered]
Hi,

Love the code server!

I was wondering if there is way to be able to read local storage through extensions installed in the code-server?

Apologies if this has been asked before, I might have missed it.

**Comment by vital987** [comment]
[SFTP](https://marketplace.visualstudio.com/items?itemName=liximomo.sftp) can help.

**Comment by code-asher** [comment, marked as answer]
Nope, extensions don't have access to the browser's local storage or cookies.

**Reply by vital987** [reply]
Confused between browser's local storage & local disk storage🙄

**Reply by code-asher** [reply]
The way I understand it is that browser's local storage is in the browser itself (on the client) and "local" disk storage (in relation to the extension) is on the remote machine (where code-server is running).

I figured they meant browser storage and not disk storage because they also mentioned cookies but I might be wrong!!

**Reply by vital987** [reply]
I think he might've said local disk storage for editing the files present on local machine.

**Reply by hkdeman** [reply]
I was looking for the browser's local storage @vital987 I think it is not yet possible. Thank you for your help guys! Much appreciated :)
